src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -0,0 +1,880 @@
+            Assert.Equal(2, fieldTokens.Length);
+            foreach (var fieldToken in fieldTokens)
+            {
+                AssertEx.NotNull(fieldToken.Parent);
+                var info = model.GetSymbolInfo(fieldToken.Parent);
+                Assert.Empty(info.CandidateSymbols);
+                Assert.False(info.IsEmpty);
+                Assert.Equal("<P1>k__BackingField", info.Symbol!.Name);
+            }
+        }
+
+        [Theory]
+        [InlineData("get { return field; } set => _ = value;")] // body getter
+        [InlineData("set => _ = value; get { return field; }")] // body getter
+        [InlineData("get => field; set { _ = value; }")] // expression bodied getter
+        [InlineData("set { _ = value; } get => field;")] // expression bodied getter
+        [InlineData("get => 0; set { field = value; }")] // body setter
+        [InlineData("set { field = value; } get => 0;")] // body setter
+        [InlineData("get { return 0; } set => field = value;")] // expression bodied setter
+        [InlineData("set => field = value; get { return 0; }")] // expression bodied setter
+        public void TestSemanticModel_GetterAndSetter_OnlyOneAccessorUsesKeyword(string accessor)
+        {
+            var comp = CreateCompilation($@"
+public class C
+{{
+    public int P1 {{ {accessor} }}
+}}
+");
+            comp.VerifyDiagnostics();
+            var tree = comp.SyntaxTrees[0];
+            var model = comp.GetSemanticModel(tree);
+
+            var fieldToken = tree.GetRoot().DescendantTokens().Where(t => t.ContextualKind() == SyntaxKind.FieldKeyword).Single();
+            AssertEx.NotNull(fieldToken.Parent);
+            var info = model.GetSymbolInfo(fieldToken.Parent);
+            Assert.Empty(info.CandidateSymbols);
+            Assert.False(info.IsEmpty);
+            Assert.Equal("<P1>k__BackingField", info.Symbol!.Name);
+        }
+
+        [Fact]
+        public void TestGetFieldsToEmit()
+        {
+            var comp = CreateCompilation(@"
+public class C
+{
+    public string P { set => field = value; }
+}
+");
+            for (int i = 0; i < 3; i++)
+            {
+                var fields = ((SourceMemberContainerTypeSymbol)comp.GetTypeByMetadataName("C")!).GetFieldsToEmit().ToArray();
+                Assert.Equal(1, fields.Length);
+                Assert.Equal("<P>k__BackingField", fields[0].Name);
+            }
+        }
+
+        [Fact]
+        public void TestGetFieldsToEmit_Initializer()
+        {
+            var comp = CreateCompilation(@"
+public class C
+{
+    public string P => field;
+}
+");
+            for (int i = 0; i < 3; i++)
+            {
+                var fields = ((SourceMemberContainerTypeSymbol)comp.GetTypeByMetadataName("C")!).GetFieldsToEmit().ToArray();
+                Assert.Equal(1, fields.Length);
+                Assert.Equal("<P>k__BackingField", fields[0].Name);
+            }
+        }
+
+        [Fact(Skip = "PROTOTYPE(semi-auto-props): Cycle..")]
```

Comment:
>"PROTOTYPE(semi-auto-props): Cycle..") [](http://example.com/codeflow?start=21&length=38)

I expect this test to pass in the scoped down PR.

Reply:
Fixed.